lib/internal/errors.js

```diff
@@ -246,6 +246,10 @@ E('ERR_OUTOFMEMORY', 'Out of memory');
 E('ERR_STDERR_CLOSE', 'process.stderr cannot be closed');
 E('ERR_STDOUT_CLOSE', 'process.stdout cannot be closed');
 E('ERR_STREAM_WRAP', 'Stream has StringDecoder set or is in objectMode');
+E('ERR_STREAM_UNSHIFT_AFTER_END_EVENT', 'stream.unshift() after end event');
+E('ERR_STREAM_PUSH_AFTER_EOF', 'stream.push() after EOF');
+E('ERR_UNDERSCORE_READ_NOT_IMPLEMENTED', '_read() is not implemented');
+E('ERR_END_READABLE_CALLED_ON_NONEMPTY_STREAM', 'endReadable() called on a non-empty stream');
```

Comment:
Can you please put this in alphabetical order :-) thank you!
Also, I believe these need line wrapped at 80 chars. Run `make lint` to check